governance/votes/Executive vote - September 5, 2024.md

```diff
@@ -0,0 +1,71 @@
+---
+title: Template - [Executive Vote] Spark Proxy Spell and Launch Project Funding - September 5, 2024
+summary: Trigger Spark Proxy Spell and transfer DAI and MKR to Launch Project. 
+date: 2024-09-05T00:00:00.000Z
+address: "$spell_address"
+
+---
+# [Executive Proposal] Spark Proxy Spell and Launch Project Funding - September 5, 2024
+
+The Governance Facilitators, Dewiz, and Sidestream have placed an executive proposal into the voting system. MKR Holders should vote for this proposal if they support the following alterations to the Maker Protocol.
+
+If you are new to voting in the Maker Protocol, please see the [voting guide](https://manual.makerdao.com/governance/voting-in-makerdao/on-chain-governance) to learn how voting works.
+
+---
+
+## Executive Summary
+
+If this executive proposal passes, the following **actions** will occur within the Maker Protocol:
+- A Spark Proxy Spell will be triggered at [0x668C84584Ef8EeEd6BFb4FFB2a4Fa03231F8b241](https://etherscan.io/address/0x668C84584Ef8EeEd6BFb4FFB2a4Fa03231F8b241).
+- **9,535,994 DAI** and **2,630 MKR** will be transferred to the Launch Project.
+
+
+**Voting for this executive proposal will place your MKR in support of the actions outlined above.**
+
+Unless otherwise noted, the actions listed above are subject to the [GSM Pause Delay](https://manual.makerdao.com/parameter-index/core/param-gsm-pause-delay). This means that if this executive proposal passes, the changes and additions listed above will only become active in the Maker Protocol after the GSM Pause Delay has expired. The GSM Pause Delay is currently set to [**16 hours**](https://mips.makerdao.com/mips/details/MIP113#10-1-1a).
```

Comment:
This increased to 30 hours in the most recent spell

```suggestion
Unless otherwise noted, the actions listed above are subject to the [GSM Pause Delay](https://manual.makerdao.com/parameter-index/core/param-gsm-pause-delay). This means that if this executive proposal passes, the changes and additions listed above will only become active in the Maker Protocol after the GSM Pause Delay has expired. The GSM Pause Delay is currently set to [**30 hours**](https://mips.makerdao.com/mips/details/MIP113#10-1-1a).
```

Reply:
Thank you, I will put in a PR to update the MIPs Repo as well.

Reply:
It is included in this PR: https://github.com/makerdao/mips/pull/1132